PlayCover/Views/Sidebar Views/AppFolderView.swift

```diff
@@ -0,0 +1,268 @@
+                        folder.apps.contains($0.info.bundleIdentifier)
+                    },
+                                      selectedBackgroundColor: $selectedBackgroundColor,
+                                      selectedTextColor: $selectedTextColor,
+                                      selected: $selected,
+                                      isList: $isList,
+                                      gridLayout: gridLayout)
+                }
+                .onTapGesture {
+                    selected = nil
+                }
+            } else {
+                VStack {
+                    Text("playapp.noSources.title")
+                        .font(.title)
+                        .padding(.bottom, 2)
+                    Text("playapp.noSources.subtitle")
+                        .font(.subheadline)
+                        .foregroundColor(.secondary)
+                    Button("playapp.importIPA") {
+                        if installVM.inProgress {
+                            Log.shared.error(PlayCoverError.waitInstallation)
+                        } else if downloadVM.inProgress {
+                            Log.shared.error(PlayCoverError.waitDownload)
+                        } else {
+                            selectFile()
+                        }
+                    }
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+            }
+        }
+        .navigationTitle("sidebar.appLibrary")
+        .navigationSubtitle(folder.name)
+        .toolbar {
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    if installVM.inProgress {
+                        Log.shared.error(PlayCoverError.waitInstallation)
+                    } else if downloadVM.inProgress {
+                        Log.shared.error(PlayCoverError.waitDownload)
+                    } else {
+                        selectFile()
+                    }
+                } label: {
+                    Image(systemName: "plus.circle")
+                        .help("playapp.add")
+                }
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Spacer()
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    showSettings.toggle()
+                } label: {
+                    Image(systemName: "gear")
+                }
+                .disabled(selected == nil)
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Picker("Grid View Layout", selection: $isList) {
+                    Image(systemName: "square.grid.2x2")
+                        .tag(false)
+                    Image(systemName: "list.bullet")
+                        .tag(true)
+                }.pickerStyle(.segmented)
+            }
+            ToolbarItem(placement: .primaryAction) {
+                Button {
+                    addSheetApps.toggle()
```

Comment:
I am encountering a bug where if I edit the apps inside the folder, click `Ok`, and then switch back to the `App Library`, the library will be empty.